packages/hooks/src/useIsAccountStatusPresent.ts

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useStore } from '@deriv/stores';
+
+const AccountStatusList = [
```

Comment:
can we move this to a util file instead of this hook?
@farzin-deriv can we import the list from shared or util package here?

Reply:
@matin-deriv What would be the benefit of it? 🤔 

> can we import the list from shared or util package here?

From `@deriv/utils` yes but from `@deriv/shared` no 👀